.github/workflows/docbuild.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       - name: Build documentation
         working-directory: ncs/nrf
         run: |
-          cmake -GNinja -Bdoc/_build -Sdoc -DSPHINXOPTS_EXTRA="-W"
+          cmake -GNinja -Bdoc/_build -Sdoc
```

Comment:
-1 on doing this, fix warnings instead

Reply:
If you see the [commit log](https://github.com/nrfconnect/sdk-nrf/pull/11372/commits/1fbb56f0bc1ff64f61278e61fbe024061ea7bca6), this was a temp change to generate a shareable link, not intended as a final fix.